docs/index.html

```diff
@@ -29,6 +33,133 @@
       hideTestRequestButton: true,
     })
   </script>
+
+<!-- shift image and buttons --!>
```

Comment:
Can this commented out block be removed completely? It's useless data being sent to the customer every time they load the page if we're not using this code 🙏